sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -3042,6 +3053,8 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
 
   lazy val maxNumOrcFilesParallel: Int = get(ORC_MULTITHREAD_READ_MAX_NUM_FILES_PARALLEL)
 
+  lazy val testOrcStripeSizeRows: Option[Integer] = get(TEST_ORC_STRIPE_SIZE_ROWS)
```

Comment:
NIT:  Personally `Integer` is not required for this case here, since null is not allowed.
```suggestion
  lazy val testOrcStripeSizeRows: Option[Int] = get(TEST_ORC_STRIPE_SIZE_ROWS)
```